ql/src/java/org/apache/hadoop/hive/ql/security/authorization/plugin/metastore/events/CreateTableEvent.java

```diff
@@ -79,6 +83,24 @@ private List<HivePrivilegeObject> getOutputHObjs() {
     Database                  database = event.getDatabase();
     String                    uri   = getSdLocation(table.getSd());
 
+    // if the table is an iceberg storagehandler based table , need storageuri as hive privilege objects
+    // format: storagehandler type + cluster + location
+    LOG.debug("<== CreateTableEvent.getOutputHObjs(): ret={}", ret);
+    if (table.getParameters().containsKey(hive_metastoreConstants.META_TABLE_STORAGE)) {
+      Configuration conf = new Configuration();
```

Comment:
do we need to create the `conf` every time? how about initializing it from `preEventContext.getHandler().getConf()`

Reply:
Creating the `conf` is a waste of resources, and we don't have any reference to this instance, I'd prefer to remove it if there is no specific reason or use.

Reply:
Acknowledged.